src/main/java/com/blackduck/integration/sca/upload/client/uploaders/UploaderFactory.java

```diff
@@ -29,15 +32,20 @@
  */
```

Comment:
Add `@see BdbaUploader` and `@see ScassUploader` to this section of the javadoc insert them in alphabetical order after ArtifactsUploader.

Reply:
Added.